docs/common/registration.md

```diff
@@ -86,14 +86,23 @@ host to avoid any duplicate registrations:
 
 ### New resources ###
 
+Before registering a new resource, make sure that its FQDN is not [already registered](#searching for resources).
+
 To register a new resource, follow the instructions below:
 
-1. If you haven't already, verify that the FQDN of your resource is not [already registered](#searching-for-resources)
+1. Find the facility, site, and resource group for your resource.
```

Comment:
Note that this removes the instruction to verify the FQDN is not already registered.  Is this step no longer relevant?  (My impression is it's still relevant.)